flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -573,6 +574,20 @@ public static int getRunningContainers() {
 		return count;
 	}
 
+	public void waitUntilApplicationFinished(ApplicationId applicationId, int timeout) throws Exception {
```

Comment:
I would suggest to rename `timeout` to `timeoutSec` or better make it of `Duration` type instead of `int`.
Then also a private static final constant can be of the `Duration` type in the previous comment.

Reply:
the method can be also package private

Reply:
OK, we could make it public when tests in other package need it.